log4j-core/src/test/java/org/apache/logging/log4j/core/ExtendedLevelTest.java

```diff
@@ -22,6 +22,7 @@
 import org.apache.logging.log4j.junit.LoggerContextRule;
 import org.apache.logging.log4j.test.ExtendedLevels;
 import org.apache.logging.log4j.test.appender.ListAppender;
+
```

Comment:
We don't require whitespace here. We don't disallow it either, but not sure what the point is.